codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/HttpProtocolUnitTestGenerator.java

```diff
@@ -81,11 +81,11 @@ protected HttpProtocolUnitTestGenerator(Builder<T> builder) {
 
         opSymbol = symbolProvider.toSymbol(operation);
 
-        inputShape = model.expectShape(operation.getInput()
+        inputShape = (StructureShape) model.expectShape(operation.getInput()
```

Comment:
Saw casting being done at several places. You could have used `model.expectShape(shapeID, StructureShape.Class)` instead